packages/components/pie-modal/src/modal.scss

```diff
@@ -345,4 +349,12 @@
             opacity: 0;
         }
     }
+
+    // removes the scroll shadow effect if "background-attachment" isn't supported.
+    // @supports (background-attachment: local) isn't used as Safari 14 recognizes the property but it has no effect
+    @supports not (aspect-ratio: 1 / 1) {
```

Comment:
Would add a comment explaining why the unrelated `aspect-ratio` is used to workaround Safari 14 detection.

Reply:
`aspect-ratio` is only supported in Safari>15, thus, we can use it as an alternative.